CONTRIBUTING.md

```diff
@@ -371,6 +371,10 @@ guidelines for the community as a whole:
   used to ping maintainers to review a pull request, a proposal or an
   issue.
 
+The governance of this Moby project is under the [Moby Technical Steering Committee](https://github.com/moby/tsc).
```

Comment:
"this Moby project"? As opposed to all the other ones?

Also you need to introduce the acronym TSC somewhere (wouldn't want people to think you switched to talking about [Tractor Supply Company](https://www.tractorsupply.com/) so why not do it in the link text?

Reply:
🚜

Reply:
Thanks @mistyhacks for the review! I believe I dealt with both comments; PTAL! (The weird construct with "this Moby project" was an attempt to reference that many Moby projects (7 of them?) are under the TSC governance, not just the moby/moby repo, but definitely didn't read well anyway) :)